session/session_fail_test.go

```diff
@@ -102,7 +102,8 @@ func (s *testSessionSuite) TestGetTSFailDirtyStateInretry(c *C) {
 	tk.MustExec("create table t (id int)")
 
 	c.Assert(failpoint.Enable("github.com/pingcap/tidb/session/mockCommitError", `return(true)`), IsNil)
-	c.Assert(failpoint.Enable("github.com/pingcap/tidb/store/tikv/mockGetTSErrorInRetry", `return(true)`), IsNil)
+	c.Assert(failpoint.Enable("github.com/pingcap/tidb/store/tikv/mockGetTSErrorInRetry",
+		`1*return(true)->return(false)`), IsNil)
```

Comment:
This modification is according to 

https://github.com/pingcap/failpoint/blob/53525bdd7953438b0255d33962bc1fe6b1d57727/terms_test.go#L43

Reply:
Is this mean return false just once? Could you add comments to explain what is the purpose of 1?

Reply:
It means return true once then all falses. This case wants to mock PD timeout error could recover automatically by the retry mechanism. I'll add some comments.